### PR TITLE
Upgrade v1beta1 to v1beta2

### PR DESCRIPTION
**Pull request by Praveenrajmani**
Adds the following DriveStatus fields

- UUID
- PartitionUUID
- MajorNumber
- MinorNumber

### CODE REVIEW

**Comment by Praveenrajmani** [comment]
Blocked on https://github.com/minio/direct-csi/pull/246

File: pkg/apis/direct.csi.min.io/v1beta2/types.go [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+	// +optional
+	Topology map[string]string `json:"topology,omitempty"`
+	// +optional
+	AccessTier AccessTier `json:"accessTier,omitempty"`
+	// +optional
+	// +k8s:conversion-gen=false
+	HostPath string `json:"hostPath,omitempty"`
+	// +optional
+	// +k8s:conversion-gen=false
+	UUID string `json:"uuid,omitempty"`
```
Comment (Praveenrajmani) [inline pkg/apis/direct.csi.min.io/v1beta2/types.go:104]: We can call it as FilesystemUUID. Will rename.

**Comment by wlan0** [comment]
LGTM

File: pkg/apis/direct.csi.min.io/v1beta2/types.go [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+	PhysicalBlockSize int64 `json:"physicalBlockSize,omitempty"`
+	// +optional
+	LogicalBlockSize int64 `json:"logicalBlockSize,omitempty"`
+	// +optional
+	Topology map[string]string `json:"topology,omitempty"`
+	// +optional
+	AccessTier AccessTier `json:"accessTier,omitempty"`
+	// +optional
+	// +k8s:conversion-gen=false
+	HostPath string `json:"hostPath,omitempty"`
```
Comment (wlan0) [inline pkg/apis/direct.csi.min.io/v1beta2/types.go:101]: did we decide to rename this to `CurrentDiskName`/`CurrentDiskPath`?

**Review by wlan0 — APPROVED** [review]

**Pull request merged**